src/app/CMakeLists.txt

```diff
@@ -161,10 +161,12 @@ endif (GUI AND WIN32)
 target_link_libraries(${QBT_TARGET_NAME} ${QBT_TARGET_LIBRARIES} QtSingleApplication::QtSingleApplication)
 
 if (APPLE)
-    set(qbt_BUNDLE_NAME "${CMAKE_PROJECT_NAME}")
+    set(qbt_BUNDLE_NAME "${QBT_TARGET_NAME}")
+    execute_process(COMMAND bash -c "sed -e \"s,@EXECUTABLE@,${qbt_BUNDLE_NAME},g\" ${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist > ${qBittorrent_BINARY_DIR}/Info.plist.in")
```

Comment:
Thanks for the good advice.

> But does it make sense to create a bundle with console application at all?

If @sledgehammer999 decides to distribute macOS version of nox, yes.
Because qbittorrent-nox does not run without bundling in macOS.

But like I said before, I doubt there are needs for nox for macOS.  :smile: